src/components/abstractComponent.ts

```diff
@@ -410,35 +410,57 @@ export module Component {
       return this._fixedHeightFlag;
     }
 
-    /**
-     * Merges this Component with another Component, returning a
-     * ComponentGroup. This is used to layer Components on top of each other.
-     *
-     * There are four cases:
-     * Component + Component: Returns a ComponentGroup with both components inside it.
-     * ComponentGroup + Component: Returns the ComponentGroup with the Component appended.
-     * Component + ComponentGroup: Returns the ComponentGroup with the Component prepended.
-     * ComponentGroup + ComponentGroup: Returns a new ComponentGroup with two ComponentGroups inside it.
-     *
-     * @param {Component} c The component to merge in.
-     * @returns {ComponentGroup} The relevant ComponentGroup out of the above four cases.
-     */
-    public merge(c: AbstractComponent): Component.Group {
+    public _merge(c: AbstractComponent, below: boolean): Component.Group {
       var cg: Component.Group;
       if (this._isSetup || this._isAnchored) {
         throw new Error("Can't presently merge a component that's already been anchored");
       }
       if (Plottable.Component.Group.prototype.isPrototypeOf(c)) {
         cg = (<Plottable.Component.Group> c);
-        cg._addComponent(this, true);
+        cg._addComponent(this, below);
         return cg;
       } else {
-        cg = new Plottable.Component.Group([this, c]);
+        var mergedComponents = below ? [this, c] : [c, this];
+        cg = new Plottable.Component.Group(mergedComponents);
         return cg;
       }
     }
 
     /**
+     * Merges this Component below another Component, returning a
```

Comment:
Wait, when we have `a.above(b)`, does `a` wind up above `b`, or below it? This description suggests that `a` winds up below `b`, which seems counterintuitive.